packages/core/src/scss/components/checkout/checkoutSteps/_checkoutSteps.scss

```diff
@@ -19,6 +19,7 @@
 
 .checkout-step {
     border-bottom: container("border");
+    counter-increment: $checkoutStep-counter;
```

Comment:
It turns out that increasing the counter with `.checkout-step` creates a more consistent cross-browser counter result.